File: library/src/main/java/com/thefuntasty/tasting/TastingException.java
package com.thefuntasty.tasting;

public class TastingException extends RuntimeException {
	public TastingException(String message) {
		super(message);
	}

	public TastingException(String message, Throwable cause) {
		super(message, cause);
	}

	public TastingException(Throwable cause) {
		super(cause);
	}
}

File: library/src/main/java/com/thefuntasty/tasting/TastingException.kt
package com.thefuntasty.tasting

import java.lang.RuntimeException

class TastingException : RuntimeException {
    constructor(message: String) : super(message)

    constructor(message: String, cause: Throwable) : super(message, cause)

    constructor(cause: Throwable) : super(cause)
}
